.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should these .idea files be part of the Pull Request?

Reply:
Thanks, they should not. I removed and added .idea to the .gitignore